doc/classes/CPUParticles2D.xml

```diff
@@ -29,183 +29,183 @@
+		<member name="emission_shape" type="int" setter="set_emission_shape" getter="get_emission_shape" enum="CPUParticles2D.EmissionShape" default="0">
 			Particles will be emitted inside this region. See [enum EmissionShape] for possible values. Default value: [constant EMISSION_SHAPE_POINT].
 		</member>
-		<member name="emission_sphere_radius" type="float" setter="set_emission_sphere_radius" getter="get_emission_sphere_radius">
+		<member name="emission_sphere_radius" type="float" setter="set_emission_sphere_radius" getter="get_emission_sphere_radius" default="null">
 			The sphere's radius if [member emission_shape] is set to [constant EMISSION_SHAPE_SPHERE].
 		</member>
-		<member name="emitting" type="bool" setter="set_emitting" getter="is_emitting">
+		<member name="emitting" type="bool" setter="set_emitting" getter="is_emitting" default="true">
 			If [code]true[/code], particles are being emitted. Default value: [code]true[/code].
 		</member>
-		<member name="explosiveness" type="float" setter="set_explosiveness_ratio" getter="get_explosiveness_ratio">
+		<member name="explosiveness" type="float" setter="set_explosiveness_ratio" getter="get_explosiveness_ratio" default="0.0">
 			How rapidly particles in an emission cycle are emitted. If greater than [code]0[/code], there will be a gap in emissions before the next cycle begins. Default value: [code]0[/code].
 		</member>
-		<member name="fixed_fps" type="int" setter="set_fixed_fps" getter="get_fixed_fps">
+		<member name="fixed_fps" type="int" setter="set_fixed_fps" getter="get_fixed_fps" default="0">
 			The particle system's frame rate is fixed to a value. For instance, changing the value to 2 will make the particles render at 2 frames per second. Note this does not slow down the simulation of the particle system itself.
 		</member>
-		<member name="flag_align_y" type="bool" setter="set_particle_flag" getter="get_particle_flag">
+		<member name="flag_align_y" type="bool" setter="set_particle_flag" getter="get_particle_flag" default="false">
 			Align Y axis of particle with the direction of its velocity.
 		</member>
-		<member name="flatness" type="float" setter="set_flatness" getter="get_flatness">
+		<member name="flatness" type="float" setter="set_flatness" getter="get_flatness" default="0.0">
 		</member>
-		<member name="fract_delta" type="bool" setter="set_fractional_delta" getter="get_fractional_delta">
+		<member name="fract_delta" type="bool" setter="set_fractional_delta" getter="get_fractional_delta" default="true">
 			If [code]true[/code], results in fractional delta calculation which has a smoother particles display effect. Default value: [code]true[/code].
 		</member>
-		<member name="gravity" type="Vector2" setter="set_gravity" getter="get_gravity">
+		<member name="gravity" type="Vector2" setter="set_gravity" getter="get_gravity" default="Vector2( 0, 98.8 )">
```

Comment:
Note to self: review this, doesn't seem to match 10 times the standard gravity: https://en.wikipedia.org/wiki/Standard_gravity

Reply:
I always set default gravity to `Vector2(0, 700)`, higher than that the objects tend to penetrate colliders more often, but that could be solved with higher linear damp perhaps.

Reply:
Since 2D units are pixels, there's no relation to m/s^2 anyway. 98 is just a really low value.

Reply:
Well, I didn't say it should change, just that 98.8 != 98.07 which would be 10g. The 2D physics engine considers 10px = 1m by default, hence this default value.